### ISSUE TITLE
Program interrupt when multi-GPU training

### ISSUE THREAD

**Issue by hxi667** [exitudio/GaitMixer#3]
Hi, it is great work! But I also needed some help. When I run train.py with multiple GPUs, (for example, the "--gpus" parameter is set to "0,1,2,3,4,5,6,7"), my program interrupts but returns no errors.  I found that the interrupts occurred in the "loss.backward()" line of code. Can you give me some advice? Thank you very much!!

**Comment by exitudio** [comment]
It may be something with the GPU environment. Have you tried with only 1 GPU and 2 GPUs?
(```export CUDA_VISIBLE_DEVICES=0```)

**Comment by hxi667** [comment]
Yes, no problem when I'm just using a gpu, I've set os.environ["CUDA_VISIBLE_DEVICES"]="0,1" , but it still doesn't work.

**Comment by exitudio** [comment]
Are you using clusters or multiprocess? The code uses [DataParallel](https://github.com/exitudio/GaitMixer/blob/main/src/train.py#L188) so it doesn't support multiprocess.

**Comment by exitudio** [comment]
You can try ```--loss_func supcon``` to see that the Triplet loss causes this problem or not.

**Comment by hxi667** [comment]
I changed the conda environment to the one used by Garph and the problem was solved! I guess it could be a certain package version that is causing the problem. Thank you again for your kind answers!
